apps/api/src/report/constants/report-four-format.ts

```diff
@@ -0,0 +1,31 @@
+import { STATUS } from '@ien/common';
+
+export const REPORT_FOUR_STEPS = [
```

Comment:
this is the same as `LIC_REG_STAGE` no? couldn't we just re-order that to match this? doesn't look like it's currently being used for anything that requires a specific order

Reply:
That makes sense, I'll give it a shot.

Reply:
I would leave a comment that the order should not be changed on this. Just in case